docs/content/node-operation/full-observer-node.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: Full Observer Node
+sidebar_title: Observer Node
+description: Running your own full observer node
+---
+
+A full observer node is an unstaked access node that follows the flow consensus and serves the Access API using local data. SendTransaction and script execution requests are forwarded to an upstream staked access node. This allows anyone to setup an access node for local queries without the need for staking.
+
+The node does the following:
+* Maintains state by following a staked access node
+* Verifies and stores blocks received from its upstream staked access node
+* Serves the Access API
+    * Get requests are processed using the node's local state
+    * SendTransaction and script executions are forwarded to the configured staked access node
+* Participates in the LibP2P network for unstaked nodes
+
+The node does not:
+* Sync the full blockchain history. For archival functionality, see [DLP](#data-provisioning-service).
```

Comment:
Is this a typo?

Reply:
yea it should be `DPS`. I removed that in the latest revision.